.vuepress/components/JupyterNotebook.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class="nb-notebook" v-html="notebookHTML"></div>
+</template>
+<script>
+import "!!script-loader!notebookjs";
+import "./ipython-style.styl";
+import "prismjs/themes/prism.css";
+import Prism from "prismjs/prism";
+import axios from "axios";
+import ansi_up from "ansi_up";
+import markdown from "marked";
```

Comment:
why not nb.markdown = require('marked');?

Reply:
We don't want to mix import and require; everything should consistently use modern es6+ imports where possible.

Reply:
We already kinda mix them [here](https://github.com/galaxyproject/SARS-CoV-2/pull/87/files#diff-a086da24c351e6cb70ec6f0f013c2d81R18). But yes, I guess I got what you mean.

P.S. Sorry for computed property, I needed to use ```$nextTick``` in https://github.com/galaxyproject/SARS-CoV-2/pull/110, but after rework in https://github.com/galaxyproject/SARS-CoV-2/pull/111 I probably should have swapped it back to computed.

Reply:
@OlegZharkov Right, and I'm actually working on fixing that one now.

Instead of nextTick and shoving it into `mounted()`, moving the highlight logic to a different vue lifecycle hook (`updated`) would have been better under that approach.

Reply:
@dannon oh yeah. That's a good idea :+1: